agent/src/test/java/org/openjdk/jmc/agent/test/TestEmitOnlyOnException.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2018, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Nit: I think it's fine to set copyright to only 2020.

Reply:
Fixed, thanks!